web/themes/custom/pineapple/templates/content/node--page--full.html.twig

```diff
@@ -106,7 +106,7 @@
       </div>
     </footer>
   {% endif %}
-  <div{{ content_attributes.addClass('node__content', 'u-o-2') }}>
+  <div{{ content_attributes.addClass('node__content', 'u-o-4') }}>
```

Comment:
Update node__content class to something else

Reply:
This doesn't belong in this PR. It is an absolute mess. This branch is for clickable card only and any minor bugs. These pull requests and commits need to be presentable from now on. If you have comments that don't apply to the current branch, please use notion to communicate them to me.